packages/terra-paginator/package.json

```diff
@@ -32,10 +32,14 @@
   "dependencies": {
     "classnames": "^2.2.5",
     "prop-types": "^15.5.8",
+    "react-router": "^4.2.0",
```

Comment:
Adding react-router as a dependency here feels weird.

Reply:
Agreed, but the examples would need to be re-worked to remove the dependency.

Reply:
Good call, I'm not sure where that came from then.